Formula/root.rb

```diff
@@ -14,12 +14,17 @@ class Root < Formula
   end
 
   depends_on "cmake" => :build
+  depends_on "davix"
   depends_on "fftw"
   depends_on "gcc" # for gfortran.
   depends_on "graphviz"
   depends_on "gsl"
+  depends_on "libxml2"
```

Comment:
Aren't `libxml2` and `ncurses` unnecessary? root seems happy to build with the system version of those.

Reply:
If using the system version is expected/possible, then these can be dropped.

Reply:
Yes the current linkage is
```
bash-3.2$ brew linkage root
System libraries:
  /System/Library/Frameworks/Accelerate.framework/Versions/A/Accelerate
  /System/Library/Frameworks/AppKit.framework/Versions/C/AppKit
  /System/Library/Frameworks/ApplicationServices.framework/Versions/A/ApplicationServices
  /System/Library/Frameworks/Cocoa.framework/Versions/A/Cocoa
  /System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation
  /System/Library/Frameworks/CoreGraphics.framework/Versions/A/CoreGraphics
  /System/Library/Frameworks/CoreText.framework/Versions/A/CoreText
  /System/Library/Frameworks/Foundation.framework/Versions/C/Foundation
  /System/Library/Frameworks/OpenGL.framework/Versions/A/OpenGL
  /System/Library/PrivateFrameworks/CoreSymbolication.framework/Versions/A/CoreSymbolication
  /usr/lib/libSystem.B.dylib
  /usr/lib/libc++.1.dylib
  /usr/lib/libncurses.5.4.dylib
  /usr/lib/libobjc.A.dylib
  /usr/lib/libxml2.2.dylib
  /usr/lib/libz.1.dylib
Homebrew libraries:
  /usr/local/opt/fftw/lib/libfftw3.3.dylib (fftw)
  /usr/local/lib/gcc/8/libgcc_s.1.dylib (gcc)
  /usr/local/opt/gcc/lib/gcc/8/libgfortran.5.dylib (gcc)
  /usr/local/opt/gcc/lib/gcc/8/libquadmath.0.dylib (gcc)
  /usr/local/opt/gsl/lib/libgsl.23.dylib (gsl)
  /usr/local/opt/gsl/lib/libgslcblas.0.dylib (gsl)
  /usr/local/opt/openssl/lib/libcrypto.1.0.0.dylib (openssl)
  /usr/local/opt/openssl/lib/libssl.1.0.0.dylib (openssl)
  /usr/local/opt/pcre/lib/libpcre.1.dylib (pcre)
  /usr/local/opt/pcre/lib/libpcreposix.0.dylib (pcre)
  /usr/local/opt/python/Frameworks/Python.framework/Versions/3.6/Python (python)
  /usr/local/opt/sqlite/lib/libsqlite3.0.dylib (sqlite)
  /usr/local/opt/xrootd/lib/libXrdCl.2.dylib (xrootd)
  /usr/local/opt/xrootd/lib/libXrdClient.2.dylib (xrootd)
  /usr/local/opt/xrootd/lib/libXrdUtils.2.dylib (xrootd)
  /usr/local/opt/xz/lib/liblzma.5.dylib (xz)
Indirect dependencies with linkage:
  sqlite
Variable-referenced libraries:
  @rpath/libASImage.so
  @rpath/libCling.so
  @rpath/libCore.so
  @rpath/libEG.so
  @rpath/libFTGL.so
  @rpath/libFoam.so
  @rpath/libGLEW.so
  @rpath/libGQuartz.so
  @rpath/libGed.so
  @rpath/libGeom.so
  @rpath/libGpad.so
  @rpath/libGraf.so
  @rpath/libGraf3d.so
  @rpath/libGui.so
  @rpath/libHist.so
  @rpath/libHistFactory.so
  @rpath/libImt.so
  @rpath/libMLP.so
  @rpath/libMathCore.so
  @rpath/libMathMore.so
  @rpath/libMatrix.so
  @rpath/libMinuit.so
  @rpath/libMultiProc.so
  @rpath/libNet.so
  @rpath/libNew.so
  @rpath/libPhysics.so
  @rpath/libProof.so
  @rpath/libProofPlayer.so
  @rpath/libRGL.so
  @rpath/libRIO.so
  @rpath/libRint.so
  @rpath/libRooFit.so
  @rpath/libRooFitCore.so
  @rpath/libRooStats.so
  @rpath/libTMVA.so
  @rpath/libThread.so
  @rpath/libTree.so
  @rpath/libTreePlayer.so
  @rpath/libTreeViewer.so
  @rpath/libXMLIO.so
  @rpath/libXMLParser.so
  @rpath/libXrdProofd.so
  @rpath/libtbb.dylib
  @rpath/libtbbmalloc.dylib
bash-3.2$ 
```
using system libxml2 and ncurses.